src/CallerArgumentExpressionAttribute.cs

```diff
@@ -21,7 +21,7 @@ public CallerArgumentExpressionAttribute(string parameterName)
 
 #endif
 
-#if NETSTANDARD2_0
+#if NETSTANDARD2_0 || NETFRAMEWORK
```

Comment:
Should we change the condition to also include NS2.1?

```suggestion
#if NETSTANDARD || NETFRAMEWORK
```

Reply:
No, the `NotNullAttribute` is in netstandard2.1, just not netstandard2.0 (or netfx).

https://apisof.net/catalog/c9927a20ea8a7ffe46c456b1d55b5d70